src/plugins/data/public/utils/table_inspector_view/components/data_table.tsx

```diff
@@ -192,12 +193,25 @@ export class DataTableFormat extends Component<DataTableFormatProps, DataTableFo
 
     return (
       <EuiInMemoryTable
-        className="insDataTableFormat__table"
+        tableLayout="auto"
+        className={classNames('insDataTableFormat__table', 'eui-xScroll')}
```

Comment:
nit: I think the `classnames` dependency can be skipped:

```suggestion
        className='insDataTableFormat__table eui-xScroll'
```

Reply:
Done!